heron/spi/src/java/com/twitter/heron/spi/common/Context.java

```diff
@@ -278,4 +292,16 @@ public static String shellBinary(Config cfg) {
   public static String pythonInstanceBinary(Config cfg) {
     return cfg.getStringValue(Key.PYTHON_INSTANCE_BINARY);
   }
+
+  public static final Boolean isCleanStateCheckpoints(Config cfg) {
+    return cfg.getBooleanValue(Key.IS_CLEAN_STATEFUL_CHECKPOINTS);
+  }
+
+  @SuppressWarnings("unchecked")
+  public static final String stateBackendSandboxClassPath(Config cfg) {
```

Comment:
Do we need this? We mostly refactored out the concept of "sandbox" by introducing config value wildcards.